hadoop-hdds/docs/content/interface/OzoneFS.md

```diff
@@ -113,3 +113,50 @@ hdfs dfs -ls o3fs://bucket.volume.om-host.example.com:6789/key
 Note: Only port number from the config is used in this case, 
 whereas the host name in the config `ozone.om.address` is ignored.
 
+## Setting up the ofs
+Please add the following entry to the core-site.xml.
+
+{{< highlight xml >}}
+<property>
+  <name>fs.ofs.impl</name>
+  <value>org.apache.hadoop.fs.ozone.RootedOzoneFileSystem</value>
+</property>
+<property>
+  <name>fs.defaultFS</name>
+  <value>ofs://om-host.example.com/</value>
+</property>
+{{< /highlight >}}
+
+This will make all the volumes and buckets to be the default file system for HDFS dfs commands and register the o3fs file system type.
```

Comment:
Can we change " default file system for HDFS dfs commands " =>  " default Hadoop compatible file system"